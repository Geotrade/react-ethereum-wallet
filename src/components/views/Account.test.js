import React from 'react';
import renderer from 'react-test-renderer';

describe('title html', () => {
  it('renders title', () => {
    const tree = renderer
      .create(
        <h1>
          <strong>Accounts</strong> Overview
        </h1>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
