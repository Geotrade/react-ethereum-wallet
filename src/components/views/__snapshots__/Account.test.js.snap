// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`title html renders title 1`] = `
<h1>
  <strong>
    Accounts
  </strong>
   Overview
</h1>
`;

exports[`wallet html renders wallet description 1`] = `
Array [
  <h2>
    Wallet Contracts
</h2>,
  <p>
    These contracts are stored on the blockchain and can hold and secure Ether. They can have multiple accounts as owners and keep a full log of all transactions.
</p>,
  <div
    className="dapp-clear-fix"
/>,
]
`;
